src/test/java/core/basesyntax/service/ParserServiceImplTest.java

```diff
@@ -0,0 +1,81 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.Operation;
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import core.basesyntax.strategy.handler.BalanceOperationHandler;
+import core.basesyntax.strategy.handler.OperationHandler;
+import core.basesyntax.strategy.handler.PurchaseOperationHandler;
+import core.basesyntax.strategy.handler.ReturnOperationHandler;
+import core.basesyntax.strategy.handler.SupplyOperationHandler;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ParserServiceImplTest {
+    private static List<String> lines;
+    private static Map<Operation, OperationHandler> handlerMap;
+    private static ParserServiceImpl parserService;
+
+    @Before
+    public void setUp() {
+        lines = new ArrayList<>();
+        handlerMap = new HashMap<>();
+        handlerMap.put(Operation.BALANCE, new BalanceOperationHandler());
+        handlerMap.put(Operation.RETURN, new ReturnOperationHandler());
+        handlerMap.put(Operation.SUPPLY, new SupplyOperationHandler());
+        handlerMap.put(Operation.PURCHASE, new PurchaseOperationHandler());
+        parserService = new ParserServiceImpl();
```

Comment:
move this to BeforeClass

Reply:
i guess, here better to use @Before cause one test using empty list , and second using filled out, but i can make another empty list for parse_emptyList_ok() if u think i should still make it with @BeforeClass

Reply:
ok, i did with new empty list with @BeforeClass